tests/garage/envs/test_normalized_gym.py

```diff
@@ -35,8 +43,16 @@ def test_unflatten(self):
         for i in range(10):
             env.reset()
             for e in range(5):
-                action = env.action_space.sample()
-                next_obs, reward, done, info = env.step(action)
+                a = env.action_space.sample()
+                a_copy = a if isinstance(a, int) else a.copy()
+                next_obs, reward, done, info = env.step(a)
+
+                # Check for side effects
+                if isinstance(a, int):
```

Comment:
I will get a little picky here, but would this work for a scalar continuous action (e.g. float)?

Reply:
Good point.